manual/source/Tutorial/Algorithms/PatternMatching/IndexedPatternMatching.rst

```diff
@@ -17,34 +17,21 @@ Duration
   30 min
 
 Prerequisites
-  :ref:`tutorial-datastructures-sequences`, :ref:`tutorial-datastructures-indices`
+  :ref:`tutorial-datastructures-sequences`, :ref:`tutorial-datastructures-indices`, :ref:`tutorial-algorithms-pattern-matching-online`
 
 Overview
 --------
 
-SeqAn provides two methods for searching for a pattern in index structures.
-One method uses iterators and is similar to traversing search trees or tries.
-The tutorial :ref:`tutorial-datastructures-indices-index-iterators` explains this method in more detail.
-In this section you will learn how to find a pattern with the :dox:`Finder` interface.
+The :dox:`Finder` is an object that stores all necessary information for searching for a pattern. We have learned how to use it in :ref:`tutorial-algorithms-pattern-matching-online` tutorial.
 
-The :dox:`Finder` is an object that stores all necessary information for searching for a pattern using an index.
-The following line of code shows how the :dox:`Finder` is initialized.
+The following line of code shows how the :dox:`Finder` is initialized with an index. In this example, we search for the pattern ``ACGT``.
 
 .. includefrags:: demos/tutorial/indices/base.cpp
       :fragment: finder
 
-After initialization it is possible to use the :dox:`Finder#find` function in order to trigger a search for all occurrences of a given pattern in the underlying :dox:`String` or :dox:`StringSet`.
-In this example, we search for the pattern ``ACGT``:
-
-.. includefrags:: demos/tutorial/indices/base.cpp
-      :fragment: finder2
```

Comment:
I think finder2 is not used anywhere else, right? So we can remove the correspoding code from ``demos/tutorial/indices/base.cpp``